_layouts/home.html

```diff
@@ -127,11 +127,12 @@ <h2>Contributor Hall of Fame</h2>
                                 {% endif %}
 
                                 {% assign username = contributor[0] %}
+                                {% assign fullname = contributor[1].name %}
                                 <div class="hall-of-fame-hero">
                                     <a class="thumbnail" href="{{ site.baseurl }}/hall-of-fame#{{ username }}">
                                         <img class="img-fluid" src="https://avatars.githubusercontent.com/{{ username }}" alt="{{ username }}">
                                     </a>
-                                    <p class="name">{{ username }}</p>
+                                    <p class="name">{{ fullname }}</p>
```

Comment:
@willdurand I changed it to use the full name of contributors in the slide show, but for some people with long names, the name is wrapped over 2 lines, causing the next line of text (L147) to jump up and down, what is the best way to fix that do you think?

Reply:
reverting to the `username`? and submitting a new PR to fix this right after?

Reply:
sure, can do that, thanks :)

Reply:
I can work on the fix after but it would be great to land this PR.

Reply:
I switched it back for now, thanks @willdurand for all your help with this PR! :)